src/configs/base.ts

```diff
@@ -164,16 +164,34 @@ export const config: Config = {
   onSurface: [
     {
       name: 'border',
-      description: 'For use as a border on non-interactive elements.',
+      description: 'For use as the default border on elements.',
       light: {lightness: 60},
       dark: {lightness: 35},
       meta: {
         figmaName: 'Border/Default',
       },
     },
+    {
+      name: 'borderHovered',
+      description: 'Used for borders on hovered interactive elements',
+      light: {lightness: 65},
+      dark: {lightness: 35},
+      meta: {
+        figmaName: 'Border/Hovered',
+      },
+    },
+    {
+      name: 'borderDisabled',
+      description: 'Used for disabled borders on interactive elements',
+      light: {lightness: 85},
+      dark: {lightness: 45},
+      meta: {
+        figmaName: 'Border/Disabled',
+      },
+    },
     {
       name: 'borderSubdued',
-      description: 'For use as a subdued border on non-interactive elements.',
+      description: 'For use as a secondary border on elements.',
```

Comment:
This seems like it should still say subdued:

```suggestion
      description: 'For use as a subdued border on elements.',
```